Week2/Assignment3/src/App.js

```diff
@@ -0,0 +1,73 @@
+import React from "react";
+import ReactDOM from "react-dom";
```

Comment:
can have an empty line between libraray imports and local file imports for better readibility